src/main/resources/graphql/schema.graphqls

```diff
@@ -95,6 +95,7 @@ type Expense {
 	amount: Float
 	title: String
 	description: String
+    date: String
```

Comment:
What does this return the date as? I was looking into creating a Scalar for Date but if this does the same thing then might as well go with it.